CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+:new: **New features**
+
+- Output source maps and use minified code in examples ([PR 1152](https://github.com/nhsuk/nhsuk-frontend/pull/1152))
+
```

Comment:
Not sure this needs a changelog entry as it doesn't affect downstream services at all? (And it could be confusing?) 🤷

Reply:
Ah I guess it the npm package includes them then it's ok to mention? 

I had assumed most services import the source from the package and recompile it together with their own stuff - but I guess you could also use the precompiled versions as in Option 2: https://service-manual.nhs.uk/design-system/production